modules/end-user-guide/pages/selecting-an-in-browser-ide-for-one-new-workspace.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: PROCEDURE
+:description: Selecting an in-browser IDE for one new workspace
+:keywords: select-IDE
+:navtitle: Selecting an in-browser IDE for one new workspace
+:page-aliases: selecting-an-ide-by-using-a-url-parameter.adoc
+
+[id="selecting-an-in-browser-ide-for-one-new-workspace_{context}"]
+= Selecting an in-browser IDE for one new workspace
+
+You can select your preferred in-browser IDE when using a URL for starting a new workspace. This way, each developer using {prod-short} can start a workspace with a clone of the same project repository and yet with the personal choice of the in-browser IDE.
+
+.Procedure
+
+. Include the URL parameter for the workspace IDE in the URL for starting a new workspace. See xref:url-parameter-for-the-workspace-ide.adoc[].
+. Visit the URL in the browser. See xref:starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc[].
```

Comment:
```suggestion
. Open the URL in the browser. See xref:starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc[].
```

Reply:
Thanks for the tip, Jana. I'll keep this option for future use. However, in this case, there's some change of the URL, so the user ends up getting a different URL open up in the browser tab.